gui/src/components/modalServer.vue

```diff
@@ -1417,10 +1438,6 @@ export default {
   color: #4099ff !important;
 }
 
-.readonly {
```

Comment:
Why remove it? Because there is no reference?

Reply:
Yes, we have to keep source code clean.